src/pages/index.js

```diff
@@ -3,95 +3,157 @@ import Card from "/src/components/Card.js";
 import FormValidator from "/src/components/FormValidator.js";
 import Section from "/src/components/Section.js";
 import UserInfo from "/src/components/UserInfo.js";
+import PopupWithConfirmation from "/src/components/PopupWithConfirmation.js";
 import PopupWithForm from "/src/components/PopupWithForm.js";
 import PopupWithImage from "/src/components/PopupWithImage.js";
-import { initialCards, config } from "/src/utils/constants.js";
-
-const profileName = document.querySelector(".profile__title");
-const profileJob = document.querySelector(".profile__description");
-const profileGallery = document.querySelector(".gallery__cards");
-
-const cardTemplate = document.querySelector("#card-template").content;
-
-const editModal = document.getElementById("edit-modal");
-const newCardModal = document.getElementById("new-card-modal");
-const previewImageModal = document.querySelector("#preview-image-modal");
-
-const previewImage = previewImageModal.querySelector(".preview-image");
-const previewImageText = previewImageModal.querySelector(".image-description");
-
-const profileEditBtn = document.querySelector(".profile__edit-button");
-const newCardBtn = document.querySelector(".profile__plus-button");
-
-const formValidators = {};
-
-const enableValidation = (config) => {
-  const formList = [...document.querySelectorAll(config.formSelector)];
-  formList.forEach((form) => {
-    const validator = new FormValidator(config, form);
-    const formName = form.getAttribute("name");
+import * as constants from "/src/utils/constants.js";
+import Api from "/src/components/Api.js";
+// User Data
+const userInfo = new UserInfo({
+  userName: constants.profileName,
+  userDescription: constants.profileDescription,
+  userProfileImage: constants.profileAvatar,
+});
+// Api Instantiation
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9fbfab1b-69d6-43a9-b085-60d57b074309",
+    "Content-Type": "application/json",
+  },
+});
 
-    formValidators[formName] = validator;
-    validator.enableValidation();
-  });
-};
-const createCard = (data) => {
-  const cardElement = new Card({
-    data: data,
-    cardSelector: cardTemplate,
-    handleCardClick: () => {
-      popupImage.open(data);
+Promise.all([api.getUserInformation(), api.getInitialCards()]).then(
+  ([userData, initialCards]) => {
+    userInfo.setUserInfo(userData);
+    userInfo.setUserProfileImage(userData);
+    const apiCardSection = new Section(
+      {
+        items: initialCards,
+        renderer: addCard,
+      },
+      constants.cardGallery
+    );
+    apiCardSection.renderItems();
+  }
+);
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**